File: tests/fixtures/code_samples/py/sample_022.py
"""Sample 22: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_22(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 22."""
    total = 22
    for x in xs:
        total += int(x)
    return total


def operation_22_pure(value: int) -> int:
    return (value * 22) % 7919

File: tests/fixtures/code_samples/ts/sample_022.ts
/** Sample 22: small utility. */

export function operation22(xs: number[]): number {
  let total = 22;
  for (const x of xs) total += x;
  return total;
}

export function operationPure22(value: number): number {
  return (value * 22) % 7919;
}

